src/Components/MediaChannelComponent.ts

```diff
@@ -0,0 +1,176 @@
+import {Component} from "../Component";
+import {GuildMember, Message, MessageReaction, TextChannel, User, VoiceState} from "discord.js";
+import {ComponentNames} from "../Constants/ComponentNames";
+import {isAdmin} from "../HelperFunctions";
+import {DJMTGuild} from "../DJMTGuild";
+import {ComponentCommands} from "../Constants/ComponentCommands";
+
+/**
+ * Declare data you want to save in JSON here. This interface is used for getSaveData and
+ * afterLoadJSON, as it tells Typescript what data you're expecting to write and load.
+ */
+interface MediaComponentSave {
+    channels: Array<string>;
```

Comment:
```suggestion
    channels: string[];
```
No real difference between the two but figured it be nice to know that this is the shorthand equivalent